tests/device_tests/bitcoin/test_signtx_invalid_path.py

```diff
@@ -215,3 +219,30 @@ def attack_processor(msg):
         )
 
         btc.sign_tx(client, "Testnet", [inp1, inp2], [out1], prev_txes=TX_CACHE_MAINNET)
+
+
+@pytest.mark.skip_t1(reason="T1 only prevents using paths known to be altcoins")
+def test_invalid_path_fail_asap(client):
+    inp1 = messages.TxInputType(
+        address_n=parse_path("0"),
+        amount=4977040,
+        prev_hash=TXHASH_a5cd2a,
+        prev_index=0,
+        script_type=messages.InputScriptType.SPENDWITNESS,
+        sequence=4294967293,
+    )
+
+    out1 = messages.TxOutputType(
+        address_n=parse_path("84h/0h/0h/1/0"),
+        amount=4977040,
+        script_type=messages.OutputScriptType.PAYTOWITNESS,
+    )
+
+    with client, pytest.raises(TrezorFailure) as exc:
+        client.set_expected_responses(
```

Comment:
does this fail the right way if the expected responses don't match?

maybe a better way would be to just `try: btc.sign_tx; except: pass`, and leave only the `expected_responses` as a thing we're testing